### DISCUSSION TITLE
（事件）近日，绳网出现了一种新型诈骗

### DISCUSSION THREAD

**Discussion by WXZ1025** [share121/inter-knot#2808] [其他]
近日，绳网出现了一种新型诈骗，他们通常会在帖子中引入一些视频链接进行网络诈骗，希望朱鸢警官可以管管
![Screenshot_2024-08-18-21-28-17-846_mark via-edit](https://github.com/user-attachments/assets/267bbd88-323a-40c4-81b5-8df6b537d6e2)

**Comment by WXZ1025** [comment]
新艾利都警察局不管管吗？

**Comment by SKAIZH** [comment]
放心吧市民！我们会处理的

**Reply by WXZ1025** [reply]
感谢警官！

**Reply by 13520rygvv** [reply]
原来治安官平时没事的时候也会逛绳网吗 真的没问题吗

**Reply by hjmwcnm** [reply]
笑了

**Comment by Maple013** [comment]
滴嘟——
前新艾利都治安局治安官提醒您，保护个人隐私，谨防信息泄露，请安全使用绳网，不要轻易点击其他人发布的网页链接，以防被Never Gonna Give You Up！

**Comment by Aha-SWYH** [comment]
没事，反正我加载不出来

**Comment by aizai1173** [comment]
已经被骗到了...两次<img src="https://img-static.mihoyo.com/communityweb/upload/1b5600e9d6d16de86447348909796aa4.png" alt="安比 哭哭" width="75" >

**Comment by LuAn-afl** [comment]
滴嘟——
前新艾利都治安局治安官提醒您，保护个人隐私，谨防信息泄露，请安全使用绳网，不要公开自己的照片、身份信息等个人隐私！

**Comment by linlu0792** [comment]
是这样的，我要报警辣，朱鸢警官马上就到

**Comment by lllliufeng** [comment]
近日，绳网出现了一种[新型诈骗](https://www.bilibili.com/video/BV1GJ411x7h7)，他们通常会在帖子中引入一些[视频链接](https://www.bilibili.com/video/av170001)进行[网络诈骗](https://www.bilibili.com/video/BV1im42137GG)，希望朱鸢警官可以管管（doge

**Reply by nzj2004** [reply]
骗子，大骗子，可恶啊

**Reply by Robin-txdy** [reply]
每天上一当，当当不一样

**Comment by waerte-yang** [comment]
6

**Comment by Pheathon** [comment]
这个委托我接了

**Comment by homteam** [comment]
我已经认错了这是我的 [道歉视频]( https://www.bilibili.com/video/BV1To4y1Z7Ne/?share_source=copy_web&vd_source=1b9c9ba08818c4515231fa3984e88d43)，（在哔站）

**Reply by WXZ1025** [reply]
666不忘初心

**Reply by nzj2004** [reply]
可恶

**Comment by ybbgh** [comment]
<a href="https://www.bilibili.com/video/BV17e8ceeEGq/?spm_id_from=333.788.recommend_more_video.5&vd_source=9f1c67940ef7fee251eca4e9f4089507" target="_blank">❤点❤我❤观❤看❤邦❤布❤可❤爱❤录❤像❤</a>

**Reply by lanximuli** [reply]
居然没被骗，良心！

**Reply by Moyuuathise** [reply]
什么？情报是真的？！

**Comment by shawnho1029** [comment]
我差的四秒誰給我補阿!?

**Comment by tingyu919** [comment]
这种污染绳网环境的就该严打，希望治安局能够严厉打击罪犯，鹅，等等，为什么绳网里会有治安局的人？！！！！

**Comment by lllliufeng** [comment]
哇，又顶上来了。
\>[点我](https://www.bilibili.com/video/BV1EV411s7vu)<让更多人看到（

**Comment by koshsa** [comment]
朱警官可能不用绳网，你得去找前辈
